### DISCUSSION TITLE
Autocue override custom start next point in advanced playlists

### DISCUSSION THREAD

**Discussion by DJMic** [AzuraCast/AzuraCast#7318] [Q&A]
I don t know if there is a bug, but I have some songs with customized cue points used in advanced playlists. When these songs are played, Autocue override my cue points especially start next point.
I'm using latest stable version 0.20.1 
Thanks for your help

**Comment by BusterNeece** [comment]
@DJMic This may be fixed in the latest Rolling Release; we've done a ton of work on the AutoCue system, so I'd recommend updating to that if possible.

**Reply by DJMic** [reply]
Thanks for your answer and Congratulations to you and @Moonbase59 for autocue. It's a wonderful tool
